.parcelrc

```diff
@@ -6,5 +6,6 @@
     // Parcel uses its own transpiler
     // override the default Parcel config to exclude babel
     "*.{js,mjs,jsx,cjs,ts,tsx}": ["@parcel/transformer-js", "@parcel/transformer-react-refresh-wrap"]
-  }
+  },
+  "resolvers": ["parcel-resolver-react-profiling", "..."]
```

Comment:
what do this and the `REACT_PROFILING=true` changes below do? just want to make sure we fully understand the implications of any of these changes from a security perspective

Reply:
https://github.com/goosewobbler/parcel-resolver-react-profiling/blob/master/src/index.ts

Basically does nothing until it comes across 'react-dom' or 'scheduler/tracing'.  Then it looks at the env var, if set it replaces those modules with their profiling variants - these impact rendering performance so are turned off by default.  If the env var is not set it does nothing.

Reply:
reading through the React docs it looks like the `react-dom/profiling` is the production profiling bundle, and that profiling is always enabled by default in dev mode. have we tried this?

getting this info from here: https://reactjs.org/blog/2018/09/10/introducing-the-react-profiler.html

Reply:
I ran into a lot of issues getting Parcel to play ball here.  Feel free to try it out without the resolver though.  Additionally, I think we want to the option to use profiling in prod builds.

https://gist.github.com/bvaughn/25e6233aeb1b4f0cdb8d8366e54a3977?permalink_comment_id=4166747